### PR TITLE
Details

### PR DESCRIPTION
**Pull request by Aparnaa-k**
Created a details page.

### CODE REVIEW

File: frontend/src/components/api_datacommponent/api_data.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from 'react';
+
+const ApiDataComponent = () => {
+    const [data, setData] = useState([]);
+    
+    useEffect(() => {
+        fetch('http://localhost:9123/api/get')
+          .then(response => response.json())
+          .then(data => {
+            console.log(data)
```
Comment (yash0212) [inline frontend/src/components/api_datacommponent/api_data.jsx]: Console log can be removed from the code

File: frontend/src/components/details/details.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    position: absolute;
+    top: 50%;
+}
+
+.details_left{
+    margin-top: 450px;
+    margin-left: 235px;
+}
+
+.bookaslot2{
```
Comment (yash0212) [inline frontend/src/components/details/details.css]: Please convert the CSS selector to snake_case naming to match other selectors

File: frontend/src/components/details/details.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    padding: 2px;
+    border-radius: 10px;
+}
+
+.calender{
+    width: 80%;
+    height: 50%;
+}
+
+.bookaslot3{
```
Comment (yash0212) [inline frontend/src/components/details/details.css]: Please convert the CSS selector to snake_case naming to match other selectors

**Review by yash0212 — CHANGES_REQUESTED** [review]
Please resolve the comments and make required changes

File: frontend/src/components/api_datacommponent/api_data.jsx (hunk 1 above)
Comment (Aparnaa-k) [reply frontend/src/components/api_datacommponent/api_data.jsx]: Done.

File: frontend/src/components/details/details.css (hunk 2 above)
Comment (Aparnaa-k) [reply frontend/src/components/details/details.css]: Done.

File: frontend/src/components/details/details.css (hunk 3 above)
Comment (Aparnaa-k) [reply frontend/src/components/details/details.css]: Done.

**Review by yash0212 — APPROVED** [review]
LG

**Pull request merged**